pangeo/values.yaml

```diff
@@ -56,25 +56,52 @@ jupyterhub:
 dask-gateway:
   # dask-gateway configuration goes here
   # See https://github.com/dask/dask-gateway/blob/master/resources/helm/dask-gateway/values.yaml
+  # The values here are optimized for deployment within a jupyterhub.
   gateway:
-    clusterManager:
-      image:
-        name: pangeo/base-notebook
-        tag: 2019.09.23
+    prefix: "/services/dask-gateway"
+    auth:
+      type: jupyterhub
 
-      clusterStartTimeout: 600
-      workerStartTimeout: 600
-      worker:
+    backend:
+      scheduler:
```

Comment:
thanks for kickstarting @TomAugspurger - can you add these resources requests to match the binder:
```
    backend:
      scheduler:
        cores:
          request: 0.8
          limit: 1
        memory:
          request: 1G
          limit: 2G
```

Reply:
actually sorry. I guess these are better set here values.yaml in pangeo-cloud-federation, correct? we just want the minimum config in the helm chart itself? https://github.com/pangeo-data/pangeo-cloud-federation/blob/staging/pangeo-deploy/values.yaml#L45

Reply:
Do we think those are appropriate for anyone inheriting from this image? I leaned towards "no", and favored keeping them in the actual deployments, but I could go either way.

Reply:
yep, I agree we shouldn't set those here.